contracts/staking/PoSValidatorManager.sol

```diff
@@ -50,6 +50,14 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
         mapping(bytes32 validationID => PoSValidatorRequirements) _validatorRequirements;
         /// @notice Maps the delegationID to the delegator information.
         mapping(bytes32 delegationID => Delegator) _delegatorStakes;
+        /// @notice Maps the delegationID to pending register delegator messages.
+        mapping(bytes32 delegationID => bytes) _pendingRegisterDelegatorMessages;
+        /// @notice Maps the delegationID to pending end delegator messages.
+        mapping(bytes32 delegationID => bytes) _pendingEndDelegatorMessages;
+        /// @notice Maps the delegationID to its pending staking rewards.
+        mapping(bytes32 delegationID => uint256) _redeemableDelegatorRewards;
+        /// @notice Maps the validator owner address to its pending staking rewards.
+        mapping(address validatorOwner => uint256) _redeemableValidatorRewards;
```

Comment:
Should we key this mapping on `validationID` rather than `validatorOwner`. That may be more intuitive since a single address could have multiple active validation periods at any point in time.